.github/workflows/unit_tests.yaml

```diff
@@ -30,6 +30,12 @@ jobs:
           repository: theforeman/hammer-cli-foreman
           ref: master
           path: hammer-cli-foreman
+      - name: Get hammer_cli_foreman_remote_execution
```

Comment:
That's correct, but there is one small thing to add:
 - at `Configure local gem dependencies` add `echo "gemspec path: '../hammer_cli_foreman_remote_execution', name: 'hammer_cli_foreman_remote_execution'" >> Gemfile.local.rb`

Reply:
Yeah, I wasn't sure if we need it as well. Thanks again! :)